### PR TITLE
updating docs

### PR DESCRIPTION
**Pull request by ndurumo254**

### CODE REVIEW

**Review by KareemMAX — CHANGES_REQUESTED** [review]
I think this order is irrelevant to `ivy_tests`. For context https://github.com/unifyai/ivy/blob/b8b63b610c50e846f90e6e9ced64ad3bef5e3d6a/ivy/func_wrapper.py#L14-L32 is actually a file in `ivy` that holds global function wrapper if I understand correctly.

Anyhow I think @danielmunioz and @juliagsy might have more experience than me in that part of the docs

**Comment by danielmunioz** [comment]
Yeah, these decorators are used for the framework mostly, I don't think they have any relevance for that part of the testing docs at least
Is this for the Trello task on adding this info to the docs? If so, I believe adding this info to the [function wrapping](https://unify.ai/docs/ivy/overview/deep_dive/function_wrapping.html) section of the deep dive would be better

**Comment by danielmunioz** [comment]
Yeah, I think it'd be more useful to have this in the function wrapping section :)

**Comment by ndurumo254** [comment]
> Yeah, I think it'd be more useful to have this in the function wrapping section :)

ok

**Review by zaeemansari70 — COMMENTED** [review]
Hi @ndurumo254 , 

As @danielmunioz mentioned that you should be adding these in the function wrappers section in the deep dive :+1: 
I remember @vedpatwardhan working on something similar, just tagging him to avoid any overlap.

Thanks! :slightly_smiling_face:

File: docs/overview/deep_dive/ivy_tests.rst [hunk 1]
```diff
@@ -249,6 +249,33 @@ One of the few differences between the 4 decorators that they generate different
 
 Our test decorators actually transforms to :code:`@given` decorators at PyTets collecting time, therefore this allows us to use other **Hypothesis** decorators like, :code:`@reproduce_failure`, :code:`@settings`, :code:`@seed`.
 
+- Decorator order
```
Comment (vedpatwardhan) [inline docs/overview/deep_dive/ivy_tests.rst:252]: I think this should be placed on the Function Wrappers page of the deep dive instead (as everyone's already mentioned), as that page also has descriptions of what each decorator does

**Review by vedpatwardhan — CHANGES_REQUESTED** [review]
Hey @ndurumo254, that's certainly an important think we've missed out on adding to the deep dive, just requested a small change. Thanks 🙂

**Review by KareemMAX — CHANGES_REQUESTED** [review]
I guess you might have added `bincount` by mistake. I think it deserves a different PR instead. I won't review the decorator docs because I believe there are better people here to review than me 😅

Please delete `bincount` related code and remove me as a reviewer.

**Comment by danielmunioz** [comment]
Hey! @ndurumo254!, the info looks good just a couple of things, the "test decorators" section would be work better if it'd be right before the "Decorator Order" section, it's ok to have it under "Function wrapping" but I think it'd be clearer to have it outside "Decorator order", also there is a minor typo with the word "Pytets", and could you also remove the comment linking this PR to `bincount` too please?

Thanks!

**Comment by ndurumo254** [comment]
> Hey! @ndurumo254!, the info looks good just a couple of things, the "test decorators" section would be work better if it'd be right before the "Decorator Order" section, it's ok to have it under "Function wrapping" but I think it'd be clearer to have it outside "Decorator order", also there is a minor typo with the word "Pytets", and could you also remove the comment linking this PR to `bincount` too please?
> 
> Thanks!

have done the changes

**Comment by danielmunioz** [comment]
Yeah, last thing, I just made Decorators order a subsection for clarity's sake
Overall LGTM! Merging now

Thanks @ndurumo254!!

**Pull request merged**